src/repo.rs

```diff
@@ -1088,6 +1088,21 @@ impl Repository {
         }
     }
 
+    /// Get the blame for a single file.
+    pub fn blame_file<'a>(&self, path: &Path) -> Result<Blame<'a>, Error> {
+        let path = try!(path.into_c_string());
+        let mut raw = 0 as *mut raw::git_blame;
+
+        unsafe {
+            let mut raw_opts = mem::zeroed();
+            try_call!(raw::git_blame_init_options(&mut raw_opts,
+                                            raw::GIT_BLAME_OPTIONS_VERSION));
```

Comment:
I've been trying to expose these sorts of options as builders whenever possible, so could this have an argument of `Option<&mut BlameOptions>` or something like that? There should be some examples when dealing with diffs.

It's ok to not bind the full option API just yet, that can always come later!